### ISSUE TITLE
[filename-case] incorrect filename reporting message

### ISSUE THREAD

**Issue by JounQin** [sindresorhus/eslint-plugin-unicorn#1136]
```js
// 1_.js
```

```sh
eslint 1_.js
```

```log
/Users/JounQin/Workspaces/GitHub/eslint-mdx/1_.js
  1:1  error  Filename is not in kebab case or pascal case. Rename it to `1.js` or `1.js`  unicorn/filename-case

✖ 1 problem (1 error, 0 warnings)
```

**Comment by fisker** [comment]
I don't think it's bug, why it named `1_`?

**Comment by JounQin** [comment]
```log
Rename it to `1.js` or `1.js`
```

It is confusing.

And it is actually a virtual filename generated by `eslint` itself.
